### ISSUE TITLE
Fail to build Qt4 plugin for qt4-style-kvantum-0.12.0 package

### ISSUE THREAD

**Issue by SmartFinn** [tsujan/Kvantum#438]
Hello, @tsujan 

When I trying to build qt4-style-kvantum package build failure with the following message:

```
/<<BUILDDIR>>/qt4-style-kvantum-0.12.0+repack/style/themeconfig/ThemeConfig.cpp: In constructor ‘Kvantum::ThemeConfig::ThemeConfig(const QString&)’:
/<<BUILDDIR>>/qt4-style-kvantum-0.12.0+repack/style/themeconfig/ThemeConfig.cpp:54:27: error: ‘class QApplication’ has no member named ‘devicePixelRatio’
   54 |   const qreal dpr = qApp->devicePixelRatio();
      |                           ^~~~~~~~~~~~~~~~
make[3]: *** [style/CMakeFiles/kvantum.dir/build.make:153: style/CMakeFiles/kvantum.dir/themeconfig/ThemeConfig.cpp.o] Error 1
```

The previous version (0.11.2) was building fine.

Could you fix it or it's time to stop providing the package?

**Comment by tsujan** [comment]
I had no intention to drop Qt4; I just forgot it ;)

I added a commit to make Qt4 compilation possible again. Please upgrade and try again!

Also, please note that, for Qt4, there is no difference between older and newer Kvantum versions because there's no Qt4 development anymore. However, I'll continue supporting its compilation as far as possible.

Thanks for the report!

**Comment by SmartFinn** [comment]
@tsujan Thanks. I added the changes as a patch, and now the build was successful. Could you release the 0.12.1 version?

**Comment by tsujan** [comment]
Does it interfere with your work if add the patch to your build tree for 0.12.0? Since Qt4 is practically dead and the patch is simple, I doubt that this change could justify a new release. Moreover, git builds will have it.

**Comment by tsujan** [comment]
I mean, I could make a point release if it's urgent. Is it?

**Comment by SmartFinn** [comment]
No, it's not. I have already uploaded the source package with the patch. Thanks.

**Comment by tsujan** [comment]
OK. Thanks for your reply.

**Comment by tsujan** [comment]
I released 0.12.1 because of this and also a fix to Kvantum manager.
